src/datepicker/timepicker.js

```diff
@@ -89,9 +89,8 @@ function stringToOptions(
           {hours: twelveHours + 12, minutes},
         ];
       } else {
-        const twentyFourHours = meridiem.toLowerCase().startsWith('a')
-          ? twelveHours
-          : twelveHours + 12;
+        const twentyFourHours =
+          meridiem.toLowerCase()[0] === 'a' ? twelveHours : twelveHours + 12;
```

Comment:
Wouldn’t it be slightly faster to do the [0] before the toLowerCase()?

Reply:
maybe - however, I believe v8 will optimize for the best performance, so I'd leave things as is